### PR TITLE
[BACKLOG-5323] Unable to create a DB Connection in Schema Workbench

### PR DESCRIPTION
**Pull request by mdamour1976**

### CODE REVIEW

File: workbench/assembly_ivy.xml [hunk 1]
```diff
@@ -25,5 +25,6 @@
     <dependencies defaultconf="default->default">
         <dependency org="pentaho" name="pentaho-mondrianschemaworkbench-plugins" rev="${dependency.schemaworkbench-plugins.revision}" transitive="false" changing="true"/>
         <dependency org="pentaho" name="pdi-dataservice-client-plugin" rev="${dependency.pdi-dataservice-client-plugin.revision}" transitive="false" changing="true" />
+        <dependency org="com.google.guava" name="guava" rev="${dependency.guava.revision}"/>
```
Comment (dkincade) [inline workbench/assembly_ivy.xml]: any concern about transitive dependencies on this one?

**Comment by mdamour1976** [comment]
@dkincade included in master:
https://github.com/pentaho/mondrian/pull/559

devci:
http://devci.pentaho.com/job/6.0-snapshot/job/mondrian-schema-workbench/43/

6.0-snapshot (cerberus):
http://cerberus.pentaho.com/job/6.0-snapshot/job/mondrian-schema-workbench/419/

File: workbench/assembly_ivy.xml (hunk 1 above)
Comment (mdamour1976) [inline workbench/assembly_ivy.xml]: There are no additional jars besides guava that are in the assembly.

**Comment by dkincade** [comment]
master: https://github.com/pentaho/mondrian/pull/559
devci: http://devci.pentaho.com/job/6.0-snapshot/job/mondrian-schema-workbench/43/

**Pull request merged**